src/nav/navActions.js

```diff
@@ -103,3 +103,14 @@ export const navigateToLegal = (): NavigationAction => StackActions.push({ route
 
 export const navigateToUserStatus = (): NavigationAction =>
   StackActions.push({ routeName: 'user-status' });
+
+export const navigateToUserScreenForUserId = (userId: number) => (
+  dispatch: Dispatch,
+  getState: GetState,
+) => {
+  const state = getState();
+  const user = getUserForId(state, userId);
```

Comment:
As @gnprice noted [on CZO](https://chat.zulip.org/#narrow/stream/243-mobile-team/topic/Selector/near/815187), this would probably be better handled via the existing `getAllUsersById` selector.